connexion/decorators/parameter.py

```diff
@@ -42,7 +43,10 @@ def snake_and_shadow(name):
     :return:
     """
     snake = inflection.underscore(name)
-    if snake in builtins.__dict__.keys():
+    all_reserved = []
+    all_reserved.extend(list(builtins.__dict__.keys()))
+    all_reserved.extend(keyword.kwlist)
+    if snake in all_reserved:
```

Comment:
We can also use the `keyword.iskeyword()` function to avoid needing to build a list:

```suggestion
    if snake in builtins.__dict__ or keyword.iskeyword(snake):
```

What do you think?